backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/picking/api/PackageableQuery.java

```diff
@@ -57,6 +60,8 @@ public class PackageableQuery
 	@Nullable WarehouseId warehouseId;
 	@NonNull @Singular ImmutableSet<LocalDate> deliveryDays;
 	@Nullable LocalDate preparationDate;
+	@Builder.Default @NonNull ZonedDateTime maximumFixedPreparationDate = getZonedDateTime();
+	@Builder.Default @NonNull ZonedDateTime maximumFixedPromisedDate = getZonedDateTime();
```

Comment:
1. pls make those nullable
2. why using (oppinioned) defaults here. why not set them by the caller API?